### PR TITLE
Dm 1621

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
1621

## Description - what does this code do?
Added TimeStamps for Favorited and Adopted in the user_practices table.

## Testing done - how did you test it/steps on how can another person can test it 
1.  Login as admin and Favorite a Practice.
2.  Go to the /admin page
3.  Click on Metrics button.
4.  Verify Practice favorited and total_count is incremented 
5.  Go back to the Hompage and Unfavorite the practice you just favorited.
6.  go to the /admin page and click on Metrics button
7.  verify that favorited count is the same but the Total_count favorited is decremented by one.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -42,8 +42,8 @@ def set_dashboard_values
       }
 
       @practices_favorited_stats = {
-        favorited_this_month: UserPractice.where(created_at: @beginning_of_current_month..@end_of_current_month, favorited: true).count,
-        favorited_one_month_ago: UserPractice.where(created_at: @beginning_of_last_month..@end_of_last_month,favorited: true).count,
+        favorited_this_month: UserPractice.where(time_favorited: @beginning_of_current_month..@end_of_current_month).count,
```
Comment (emyl3) [inline app/admin/dashboard.rb:45]: Question... (sorry if I missed this in a mtg or something... 😅) but why do we want to count in the favorited count (for the current month and last month) practices that are no longer favorited by a user?? 🤔

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -313,12 +313,11 @@ def export_metrics
             "Favorited Counts"
           end
 
-          table_for practices_favorited_stats do
```
[the remark this answers is not in the document]
Comment (bradjohnson92008) [reply app/admin/dashboard.rb:316]: The Id is just for the Unit Tests.

File: app/admin/dashboard.rb [hunk 3]
```diff
@@ -355,7 +354,7 @@ def export_metrics
             "Adoption Counts"
           end
 
-          table_for practices_adoption_stats do
```
[the remark this answers is not in the document]
Comment (bradjohnson92008) [reply app/admin/dashboard.rb:358]: Same thing.. Id just for the Unit Tests

File: app/admin/dashboard.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (emyl3) [reply app/admin/dashboard.rb:316]: ahh! ok cool :) Thank you for letting me know!

File: app/admin/dashboard.rb (hunk 1 above)
Comment (bradjohnson92008) [reply app/admin/dashboard.rb:45]: bc we still want to capture that the practice was favorited in the month columns... but we do decrement the favorite total_count when the practice is UN-favorited.

Comment (emyl3) [reply app/admin/dashboard.rb:45]: ahh ok! thanks, brad! :)

**Pull request merged**